templates/admin/judge/submission/change_form.html

```diff
@@ -16,5 +19,14 @@
             <i class="fa fa-lg fa-refresh"></i>
             <span class="text">{% trans "Rejudge" %}</span>
         </a>
+        
+        {% if original.contest_or_none is not None %}
```

Comment:
Should probably not show this if the user does not have `judge.disqualify_submission`